### PR TITLE
MAYA-107484 - Time is not connected when creating a new empty USD stage

### PR DESCRIPTION
**Pull request by seando-adsk**
#### MAYA-107484 - Time is not connected when creating a new empty USD stage
* Connected time to proxy shape node

### CODE REVIEW

**Review by kxl-adsk — APPROVED** [review]

**Review by HamedSabri-adsk — APPROVED** [review]
Looking good!

File: plugin/adsk/scripts/mayaUsd_createStageFromFile.mel [hunk 1]
```diff
@@ -192,5 +192,9 @@ global proc mayaUsd_createStageFromFile() {
     if (size($result) > 0) {
         doCreateStage($result[0]);
     }
-        
+}
+
+global proc mayaUsd_createStageFromFilePath(string $usdFile)
+{
+    doCreateStage($usdFile);
```
Comment (seando-adsk) [inline plugin/adsk/scripts/mayaUsd_createStageFromFile.mel:199]: Simple global proc that allows me to do the same as proc above, but sending in a file path, skipping the file dialog.

**Review by seando-adsk — COMMENTED** [review]
I added a new test for the create stages commands. I test that the proxy node was created, selected and that the time is connected.

**Review by kxl-adsk — APPROVED** [review]
Thank you, @seando-adsk, for adding the test.

**Review by HamedSabri-adsk — APPROVED** [review]

**Pull request merged**